### ISSUE TITLE
Surface feedback surveys for Identity APIs under "Related Links" of the PowerShell cmdlets

### ISSUE THREAD

**Issue by sanana30** [Azure/autorest.powershell#805]
We'd like to surface surveys to understand user feedback on specific PowerShell cmdlets under the Get-Help "Related Links" section for Identity APIs. On each Identity cmdlet, shown below, we'd like a sentence such as "Send feedback about this cmdlet here: https://aka.ms/blablah”

Here's the table of Identity APIs and the related surveys we'd like to surface:

<html>
<body>

Priority? | Graph Explorer Endpoint | PowerShell cmdlet | Survey Link
-- | -- | -- | --
  | /conditionalAccess/policies | Get-AzureADMSConditionalAccessPolicy | https://aka.ms/caapifeedback
  |   | New-AzureADMSConditionalAccessPolicy |  
  |   | Set-AzureADMSConditionalAccessPolicy |  
  |   | Remove-AzureADMSConditionalAccessPolicy |  
  | /conditionalAccess/namedLocations | Get-AzureADMSNamedLocationPolicy |  
  |   | New-AzureADMSNamedLocationPolicy |  
  |   | Set-AzureADMSNamedLocationPolicy |  
  |   | Remove-AzureADMSNamedLocationPolicy |  
  |   |   |  
Yes | /applications | Get-AzureADApplication | https://aka.ms/applicationsAPISurvey
  |   | New-AzureADApplication |  
  |   | Set-AzureADApplication |  
  |   | Remove-AzureADApplication |  
  |   |   |  
  | /servicePrincipals | Get-AzureADServicePrincipal | https://aka.ms/servicePrincipalAPISurvey
  |   | New-AzureADServicePrincipal |  
  |   | Set-AzureADServicePrincipal |  
  |   | Remove-AzureADServicePrincipal |  
  |   |   |  
  | /servicePrincipals/{id}/appRoleAssignments | Get-AzureADServiceAppRoleAssignment | https://aka.ms/appRoleAssignmentAPISurvey
  |   | New-AzureADServiceAppRoleAssignment |  
  |   | Remove-AzureADServiceAppRoleAssignment |  
  |   |   |  
  | /onPremisesPublishingProfiles/applicationProxy/connectors | Get-AzureADApplicationProxyConnector | https://aka.ms/msgraphaadsurveyconnectors
  |   |   |  
  | /onPremisesPublishingProfiles/applicationProxy/connectorgroups | Get-AzureADApplicationProxyConnectorGroup | https://aka.ms/msgraphaadsurveyconnectorgroups
  |   |   |  
Yes | /groups | Get-AzureADGroup | https://aka.ms/GroupsAPIFeedback
  |   | New-AzureADGroup |  
  |   | Add-AzureADGroupMember |  
  |   | Remove-AzureADGroupMember |  
  |   | Remove-AzureADGroup |  
  |   |   |  
Yes | /devices | Get-AzureADDevice | https://aka.ms/deviceAPIFeedback
  |   | Set-AzureADDevice |  
  |   | Remove-AzureADDevice |  
  |   |   |  
Yes | /users | Get-AzureADUser | https://aka.ms/UsersAPIFeedback
  |   | Get-AzureADUserThumbnailPhoto |  
  |   | New-AzureADUser |  
  |   | Remove-AzureADUser |  
  |   | Set-AzureADUser |  
  |   | Set-AzureADUserThumbnailPhoto |  
  |   | Revoke-AzureADUserAllRefreshToken

</body>
</html>

**Comment by maisarissi** [comment]
Hi @sanana30
@adamedx commented [here](https://github.com/microsoftgraph/msgraph-sdk-powershell/pull/809#discussion_r684564746) about a behavior he observed that looks like we can only have one entry in the .LINKS field which is directly linked to this request.
@dingmeng-xue @dcaro do you have any thoughts? Is there such a limitation?

**Comment by dolauli** [comment]
@sanana30 Yes, it is a limitation. Currently, actually only examples in the help can be customized. We will consider to support customization for other parts in the help.

**Comment by sanana30** [comment]
Hi @maisarissi @dolauli - If we can't handle the survey link under the "Related Links" section of Get Help, is it possible for us to add a section such as "We'd like to hear from you!" and then add the survey link under that? Here's an example
![image](https://user-images.githubusercontent.com/33753308/132404522-07d9ea5f-a20f-4614-a16a-42e16ec0f8b1.png)


Adding @dhruvchand
